rpm/podman.spec

```diff
@@ -186,6 +180,17 @@ capabilities specified in user quadlets.
 It is a symlink to %{_bindir}/%{name} and execs into the `%{name}sh` container
 when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 
+%package machine
+Summary: Metapackage for setting up %{name} machine
+Requires: %{name} = %{epoch}:%{version}-%{release}
+Requires: gvisor-tap-vsock
+Requires: qemu
+Requires: virtiofsd
+
+%description machine
+This subpackage installs the dependencies for %{name} machine and also creates
+a symlink to virtiofsd in %{_libexecdir}/%{name}
```

Comment:
I don't think any user would care that it ships a symlink, that's an implementation detail.
How about:

> This subpackage installs the dependencies for "%{name} machine"; for more see https://docs.podman.io/en/latest/markdown/podman-machine.1.html

or something?

Reply:
i'll update, thanks

Reply:
done.